src/modules/settings/components/store/AnonChat.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Panel from 'rsuite/lib/Panel/index.js';
+import Toggle from 'rsuite/lib/Toggle/index.js';
+import {registerComponent, useStorageState} from '../Store.jsx';
+import {SettingIds} from '../Defaults.jsx';
+
+function AnonChat() {
+  const [value, setValue] = useStorageState(SettingIds.ANONCHAT);
+
+  return (
+    <Panel header="Anon Chat">
+      <p className="bttv-muted-text">Joins chat anonymously without appearing in the userlist</p>
+      <br />
```

Comment:
let's avoid using `br` in our react code. can we use css for padding/margins instead? it also may make more sense to inline checkboxes with their labels